### PR TITLE
Minor updates to Distributed Tracing WG charter draft

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
Minor revisions to clarify that browsers are non-goals and relationships with the web performance group.

### CODE REVIEW

File: 2022/distributed-tracing.html [hunk 1]
```diff
@@ -228,12 +230,12 @@ <h3>
...
-            <p class="draft-status"><b>Draft state:</b> <a href="https://www.w3.org/TR/2022/WD-trace-context-2-20220929/">Working Draft</a></p>
+            <p class="draft-status"><b>Draft state:</b> <a href="https://www.w3.org/TR/2023/CR-trace-context-2-20230418/">Working Draft</a></p>
             <p class="milestone"><b>Expected completion:</b> Q4 2023</p>
-            <p><b>Adopted Draft:</b> <a href='https://www.w3.org/TR/2022/WD-trace-context-2-20220929/'>2022-09-29</a></li>
-            <p><b>Exclusion Draft:</b> <a href='https://www.w3.org/TR/2022/WD-trace-context-2-20220929/'>https://www.w3.org/TR/2022/WD-trace-context-2-20220929/</a>,
+            <p><b>Adopted Draft:</b> <a href='https://www.w3.org/TR/2023/CR-trace-context-2-20230418/'>2023-04-18</a></li>
+            <p><b>Exclusion Draft:</b> <a href='https://www.w3.org/TR/2023/CR-trace-context-2-20230418/'>https://www.w3.org/TR/2023/CR-trace-context-2-20230418/</a>,
             associated <a href='https://www.w3.org/mid/cfe-10516-7be30be50df34d2d757f90ac32c4d6894b7904da@w3.org'>Call for Exclusion</a>
-on 2022-09-29, ending on 2023-02-26</i></p>
+on 2023-04-18, ending on 2023-TODO</i></p>
```
Comment (SergeyKanzhelev) [inline 2022/distributed-tracing.html:238]: @plehegar what dates should be here?

**Comment by SergeyKanzhelev** [comment]
@kalyanaj @plehegar

File: 2022/distributed-tracing.html [hunk 2]
```diff
@@ -190,6 +190,7 @@ <h3 id="out-of-scope">Out of Scope</h3>
 
           <ul class="out-of-scope">
             <li>definition of (programming) language specific APIs for performance data collection; </li>
+            <li>browsers built-in implementation of trace headers propagation; </li>
```
Comment (basti1302) [inline 2022/distributed-tracing.html]: _nitpick:_ Is this referring to the "built-in implementation ..." _of_ browsers? That is, should it have a apostrophe to indicate genitive?
```suggestion
            <li>browsers' built-in implementation of trace headers propagation; </li>
```

File: 2022/distributed-tracing.html [hunk 3]
```diff
@@ -217,7 +218,8 @@ <h3>
             <dd>
               <p>This specification is the update of <a href="https://www.w3.org/TR/trace-context-1/">Trace Context 1</a> which
                 defines headers and a value format to propagate context information that enables distributed tracing scenarios.
-                The new revision will include a mechanism to carry context information in protocol <em>responses</em>, including HTTP, thus
+                The new revisions will include an improved semantic of the header to use for scenarios requiring identifiers randomness alongside the uniquiness.
```
Comment (basti1302) [inline 2022/distributed-tracing.html]:
```suggestion
                The new revisions will include an improved semantic of the header to use for scenarios requiring the randomness of identifiers alongside their uniqueness.
```

File: 2022/distributed-tracing.html [hunk 4]
```diff
@@ -217,7 +218,8 @@ <h3>
...
+                The future versions of the specification will also include mechanism to carry context information in protocol <em>responses</em>, including HTTP, thus
```
Comment (basti1302) [inline 2022/distributed-tracing.html]:
```suggestion
                Future versions of the specification will also include mechanism to carry context information in protocol <em>responses</em>, including HTTP, thus
```

**Review by basti1302 — APPROVED** [review]
LGTM

File: 2022/distributed-tracing.html [hunk 5]
```diff
@@ -338,7 +328,8 @@ <h3 id="w3c-coordination">W3C Groups</h3>
             </dt>
             <dd>The trace context standard relates to work around Server Timing done within the Web Performance group. Going forward
                 it will make sense to define how trace context headers - specifically response headers - are related to the server timing header.
-                The trace interchange format also can be used to represent server timing data.
+                The Working Group should coordinate with the Web Performance Working Group to ensure that the two specifications are
+                aligned. Server timing headers may be used to carry the trace context information as well as the baggage data.
```
Comment (yoavweiss) [inline 2022/distributed-tracing.html]: Maybe "may be used by applications" - to clarify this in not something the browser is expected to do on its own?

**Comment by plehegar** [comment]
merging and then will do a final cleanup

**Pull request merged**